src/Microsoft.Security.Utilities.Core/DataClassification/DAT101_006_Pkcs12Certificate.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.Security.Utilities
+{
+    public class Pkcs12Certificate : RegexPattern
+    {
+        public Pkcs12Certificate()
+        {
+            Id = "DAT101/007";
```

Comment:
>DAT101 [](http://example.com/codeflow?start=18&length=6)

this needs to be a standard rule, not a `DAT` rule. :) because its intended for sure to identify sensitive data. we have not have the accuracy we want now, but hopefully we will.